### DISCUSSION TITLE
Powered Up Hub Broken

### DISCUSSION THREAD

**Discussion by maxplayz2746** [pybricks/support#809] [Q&A]
My powered up hub will not enter bootloader mode properly it will go rainbow after two pink flashes with no computer connected at all

**Comment by dlech** [comment]
Which hub are you using?

**Reply by maxplayz2746** [reply]
city hub

**Reply by dlech** [reply]
If the lights flash like that when you turn it on, it means it has bad firmware and the firmware needs to be flashed again. You can use Pybricks Code or an official LEGO app to flash the firmware again.

**Reply by maxplayz2746** [reply]
but it doesn't give me enough time to connect a device

**Reply by dlech** [reply]
If the hub is turning off, keep holding the green button to keep it turned on. Also, make sure the batteries are not low.

**Reply by maxplayz2746** [reply]
it goes rainbow when I don't connect anything though

**Comment by maxplayz2746** [comment]
it just started working thanks for the help
